env.staging

```diff
@@ -0,0 +1,83 @@
+# NODE_ENV should be one of "development" or "production"
+NODE_ENV=development
+
+# PORT is the port used by the web server
+PORT=3000
+
+# API_URL is the URL of the Telescope Web API server.  Change this to
+# pick which backend server our frontend uses.  If you are developing
+# locally, this will be localhost:{PORT}, probably http://localhost:3000.
+# Other possible values include:
+#
+#   API_URL=http://localhost:3000 (local development)
+API_URL=https://dev.telescope.cdot.systems
+#   API_URL=https://telescope.cdot.systems (our production server)
+#   API_URL=http://localhost:3000
```

Comment:
I would delete all the lines that you aren't using.  In development it's nice to have them around, but in this case, let's strip this file down to the bare essentials, and not leave any extra stuff in here we could mistakenly enable.